engine/src/main/java/org/terasology/logic/players/LocalPlayerSystem.java

```diff
@@ -162,7 +162,7 @@ private void processInput(EntityRef entity, CharacterMovementComponent character
         Quat4f viewRotation;
         switch (characterMovementComponent.mode) {
             case WALKING:
-                if (!playerCamera.isTracked()) {
+                if (!config.getRendering().isVrSupport()) {
```

Comment:
Any way to get the renderingConfig only once and assign it to a field? I noticed this class does not have a constructor. This way we don't have to fetch the rendering config every time and we can write:

`if (!renderingConfig.isVrSupport()) {`